contracts/minting/MintController.sol

```diff
@@ -58,6 +58,7 @@ contract MintController is Controller {
      * @dev sets the minterManager
      */
     function setMinterManager(address _newMinterManager) onlyOwner public returns (bool) {
+        require(_newMinterManager != address(0), "Minter manager must be a non-zero address");
```

Comment:
I'm not sure this feature is a good idea.  We may want to be able to disable the MasterMinter contract (in case of an emergency).  We have several options:

1. Change the USDC masterMinter to point to a different contract
2. Set the MasterMinter minterManager to some random address - of which 0 is the cleanest.

We should discuss this with the team.

Reply:
Remove null address check for setMintManager